plugins/Contents/tests/System/expected/test_Contents__Actions.getPageUrls_month.xml

```diff
@@ -6,23 +6,23 @@
 		<nb_hits>2</nb_hits>
 		<sum_time_spent>540</sum_time_spent>
 		<nb_hits_with_time_network>2</nb_hits_with_time_network>
-		<min_time_network>0.0330</min_time_network>
-		<max_time_network>0.0330</max_time_network>
+		<min_time_network>0.033</min_time_network>
```

Comment:
Is there a specific reason why the number formatting for those metrics changed?

Reply:
When querying the data is a string '0.0330' in day reports. When aggregating it is converted to a float, `0.033` and is stored that way in the datatable. This PR does not re-use rows in other tables when the row doesn't exist in the to be aggregated one, it uses sumRow on a new Row(), so the string value is converted to a float immediately.

Reply:
that makes sense 👍